astropy/coordinates/transformations/graph.py

```diff
@@ -0,0 +1,770 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+This module contains a general framework for defining graphs of transformations
+between coordinates, suitable for either spatial coordinates or more generalized
+coordinate systems.
+
+The fundamental idea is that each class is a node in the transformation graph,
+and transitions from one node to another are defined as functions (or methods)
+wrapped in transformation objects.
+
+This module also includes more specific transformation classes for
```

Comment:
This last part is not true for this file. I'd tend to just write a new docstring rather than copy stuff from init.  Could be as short as
```
"""
The class that implements the framework for defining graphs of transformations.
"""
```

Reply:
✅